app/precompiled.py

```diff
@@ -281,47 +311,33 @@ def _overlay_printable_areas_with_white(src_pdf):
     packet = BytesIO()
     can = canvas.Canvas(packet, pagesize=A4)
 
-    page_height = float(page.mediaBox.getHeight())
-    page_width = float(page.mediaBox.getWidth())
-
     can.setStrokeColor(white)
     can.setFillColor(white)
 
-    width = page_width - (BORDER_FROM_LEFT_OF_PAGE * mm + BORDER_FROM_RIGHT_OF_PAGE * mm)
+    width = BORDER_FROM_LEFT_OF_PAGE * mm + BORDER_FROM_RIGHT_OF_PAGE * mm
```

Comment:
do we even use this variable now?

Reply:
it was removed here https://github.com/alphagov/notifications-template-preview/pull/413/commits/3938835b4ba021202d83eaae3ad86c56b005b7cd#diff-cc3699ca803f977365cdb0525d419092L317